### ISSUE TITLE
CORS issue with https://js.stripe.com

### ISSUE THREAD

**Issue by rupesh-raj** [joshnuss/svelte-stripe-js#21]
Hi,
I am trying to use this library on a sapper project. I have created a bare implementation of PaymentElemet component and trying to do a payment.
I am running into the issue where I am getting the following error

```
Access to XMLHttpRequest at 'https://r.stripe.com/0' from 
origin 'https://js.stripe.com' has been blocked by
CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

```
POST https://r.stripe.com/0 net::ERR_FAILED 400

```

My understanding for this is that js.stripe.com based library is trying to make a call to r.stripe.com which fails as they dont allow my origin. 

For reference this is my CSP header
```
	<meta http-equiv="Content-Security-Policy" 
		content="
			img-src * 'self' data:;
			frame-src 'self' https://js.stripe.com https://hooks.stripe.com;
			connect-src * https://api.stripe.com;
			script-src 'self' 'unsafe-eval' https://js.stripe.com;
			script-src-elem * data: blob: 'unsafe-inline';
			script-src-attr 'self' data: 'unsafe-inline';
			object-src 'none'
		"
	/>
```
TIA

**Comment by joshnuss** [comment]
I believe this is a duplicate of #22, so closing.

@rupesh-raj please re-open if that's not the case.
